treesit-auto.el

```diff
@@ -81,7 +82,7 @@ automatic installation (or prompting, based on the value of
 
 (cl-defstruct treesit-auto-recipe
   "Emacs metadata for a tree-sitter language grammar."
-  lang ts-mode remap url revision source-dir cc c++)
+  lang ts-mode remap url requires revision source-dir cc c++)
```

Comment:
Minor nitpick: I'd prefer if we kept any new keywords before `:url`, so that all the keywords for `treesit-install-language-grammar` stay together